engine/lite/kafka/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/LoopUntilClosed.scala

```diff
@@ -102,9 +103,7 @@ class LoopUntilClosed(prepareSingleRunner: () => Task, waitAfterFailureDelay: Fi
           tryWithInterruptedHandle {
             status = Restarting
             Thread.sleep(delayToWait)
-          } {
-            status = Running
-          }
+          } { }
```

Comment:
after this change, we'll be switching Restarting => Running only after successful init()? I think it's ok, just wanted to make it clear

Reply:
Yep, I thought it will be simpler to make it consistent regardless of whether restarting takes place or not.